### PR TITLE
Catoc 118

### PR DESCRIPTION
**Pull request by AlokHMRC**

### CODE REVIEW

File: src/main/scala/uk/gov/hmrc/ct/computations/CP286a.scala [hunk 1]
```diff
@@ -30,7 +30,9 @@ case class CP286a
 
   override def validate(boxRetriever: ComputationsBoxRetriever): Set[CtValidation] = {
     collectErrors(
-      requiredErrorIf(value.isEmpty && boxRetriever.cpQ18().isTrue),
+      requiredErrorIf(value.isEmpty && boxRetriever.cpQ18().isTrue
+        && mayHaveNirLosses(boxRetriever) &&
+        !hasValue),
```
Comment (kastoestoramadus) [inline src/main/scala/uk/gov/hmrc/ct/computations/CP286a.scala]: If this box is new with NIR then this condition is not right.

File: src/main/scala/uk/gov/hmrc/ct/computations/CP286b.scala [hunk 2]
```diff
@@ -29,7 +29,9 @@ case class CP286b(value: Option[Int])
 
   override def validate(boxRetriever: ComputationsBoxRetriever): Set[CtValidation] = {
     collectErrors(
-      requiredErrorIf(value.isEmpty && boxRetriever.cpQ18.isTrue),
+      requiredErrorIf(value.isEmpty && boxRetriever.cpQ18().isTrue
+        && mayHaveNirLosses(boxRetriever) &&
+        !hasValue),
```
Comment (kastoestoramadus) [inline src/main/scala/uk/gov/hmrc/ct/computations/CP286b.scala]: same here

File: src/main/scala/uk/gov/hmrc/ct/computations/CP286a.scala (hunk 1 above)
Comment (AlokHMRC) [reply src/main/scala/uk/gov/hmrc/ct/computations/CP286a.scala]: This box is new with the new NI Journey.

File: src/main/scala/uk/gov/hmrc/ct/computations/CP286b.scala (hunk 2 above)
Comment (AlokHMRC) [reply src/main/scala/uk/gov/hmrc/ct/computations/CP286b.scala]: same with this one.

**Pull request merged**